feg/gateway/services/session_proxy/credit_control/gy/gy_client_test.go

```diff
@@ -398,19 +448,22 @@ func getGyGlobalConfig(apnFilter, apnOverwrite string) *gy.GyGlobalConfig {
 	}
 }
 
-func startServer(client *diameter.DiameterClientConfig, server *diameter.DiameterServerConfig, initMethod gy.InitMethod) *mock_ocs.OCSDiamServer {
+func startServer(client *diameter.DiameterClientConfig, server *diameter.DiameterServerConfig, initMethod gy.InitMethod, action gy.FinalUnitAction) *mock_ocs.OCSDiamServer {
```

Comment:
check line 464. Here you should pass a `FinalUnitInidcation` type so we can reuse it for different types of test